content/change/get_started.md

```diff
@@ -0,0 +1,20 @@
+title: Start the Change
```

Comment:
I like this page very much -- could you make sure to include it in `mkdocs.yml`, and possibly remove the title attribute?